### PR TITLE
CLDR-17535 Update likely subtags data

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-17535

Update the likely subtags with the new tool. 

Still to do (but probably after alpha)
- delete GenerateMaximalLocales and GenerateLikelyAdditions
- convert to using JSON parser
- read the wikidata each time (and fix the file name)
- update the dev instructions
- check the other tooling to make sure that the updates to languageData script and territory data make sure that the most frequent scripts and territories are at the front of the attribute values.
- check the unittest for the likely data to make sure that it checks for redundancies.

**The format has been cleaned up, and the diff will be hard to read, so look at the delta in https://github.com/unicode-org/cldr/pull/3958**

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by macchiati** [comment]
Set to draft until I resolve the errors

**Review by srl295 — DISMISSED** [review]
LGTM so far.

**Comment by macchiati** [comment]
The tests pass locally now, so crossing my fingers that the do on the server.

**Comment by macchiati** [comment]
Ping: would like to merge this this morning if possible.

File: tools/cldr-code/src/main/java/org/unicode/cldr/draft/ScriptMetadata.java [hunk 1]
```diff
@@ -139,6 +142,12 @@ public static void addNameToCode(String type, Map<String, String> hashMap) {
 
     public static final class SkipNewUnicodeException extends ICUException {}
 
+    /**
+     * Scripts that either have no known languages as yet (Cpmn) or are used for any language
+     * (Brai).
+     */
+    public static final Set<String> SCRIPTS_WITH_NO_LANGUAGES = Set.of("Brai", "Cpmn");
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/draft/ScriptMetadata.java:149]: "any language" would be good script metadata (and supplemental data).  Zyyy, Zxxx in the same boat.

**Review by srl295 — COMMENTED** [review]
looking good so far, a couplemore files to review

**Comment by macchiati** [comment]
FYI 
Base data:	3881
Minimized:	1296

**Review by srl295 — APPROVED** [review]

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/LocaleValidator.java [hunk 2]
```diff
@@ -22,6 +22,15 @@
...
     static final SupplementalDataInfo SDI = SupplementalDataInfo.getInstance();
 
+    /** For backwards compatibility, certain non-regular codes are allowed in LikelySubtags. */
+    public static final LocaleValidator.AllowedValid ALLOW_IN_LIKELY =
+            new LocaleValidator.AllowedValid(
+                    null,
+                    LstrType.region,
+                    new LocaleValidator.AllowedMatch("001|419"),
+                    LstrType.language,
+                    new LocaleValidator.AllowedMatch("und|in|iw|ji|jw|mo|tl"));
```
[the remark this answers is not in the document]
Comment (macchiati) [reply tools/cldr-code/src/main/java/org/unicode/cldr/util/LocaleValidator.java:32]: A superset is. We only want the common ones here.

**Pull request merged**